docusaurus.config.js

```diff
@@ -225,7 +229,7 @@ const config = {
               },
               {
                 label: "GitHub",
-                to: "https://github.com/facebook/docusaurus",
+                to: "https://github.com/kubearmor/kubearmor.io",
```

Comment:
```suggestion
                to: "https://github.com/kubearmor",
```
We should redirect to the org rather than specific repo.